src/test/regress/sql/alter_database_from_nonmain_db.sql

```diff
@@ -0,0 +1,294 @@
+alter database "altered_database!'2"  rename to altered_database_renamed;
+alter database altered_database_renamed  rename to "altered_database!'2";
+
+alter database "altered_database!'2" with
+    ALLOW_CONNECTIONS true
+    CONNECTION LIMIT 0
+    IS_TEMPLATE false;
+
+\c regression - - :master_port
+SELECT * FROM public.check_database_on_all_nodes($$altered_database!''2$$) ORDER BY node_type;
+
+\c regression - - :worker_1_port
+set citus.log_remote_commands = true;
+set citus.grep_remote_commands = "%ALTER DATABASE%";
+
+alter database "altered_database!'2" with
+    ALLOW_CONNECTIONS false
+    CONNECTION LIMIT 1
+    IS_TEMPLATE true;
+
+\c regression - - :master_port
+SELECT * FROM public.check_database_on_all_nodes($$altered_database!''2$$) ORDER BY node_type;
+
+\c regression - - :worker_2_port
+set citus.log_remote_commands = true;
+set citus.grep_remote_commands = "%ALTER DATABASE%";
+alter database "altered_database!'2" with
+    ALLOW_CONNECTIONS true
+    CONNECTION LIMIT 0
+    IS_TEMPLATE false;
+
+\c regression
+create role test_owner_non_main_db;
+
+\c test_alter_db_from_nonmain_db
+set citus.log_remote_commands = true;
+set citus.grep_remote_commands = "%ALTER DATABASE%";
+set citus.enable_create_database_propagation=on;
+alter database "altered_database!'2" owner to test_owner_non_main_db;
+\c regression - - :master_port
+SELECT * FROM public.check_database_on_all_nodes($$altered_database!''2$$) ORDER BY node_type;
+
+\c test_alter_db_from_nonmain_db - - :worker_1_port
+set citus.log_remote_commands = true;
+set citus.grep_remote_commands = "%ALTER DATABASE%";
+alter database "altered_database!'2" owner to CURRENT_USER;
+set default_transaction_read_only = false;
+\c regression - - :master_port
+SELECT * FROM public.check_database_on_all_nodes($$altered_database!''2$$) ORDER BY node_type;
+
+\c regression - - :worker_2_port
+set citus.log_remote_commands = true;
+set citus.grep_remote_commands = "%ALTER DATABASE%";
+alter database "altered_database!'2" owner to test_owner_non_main_db;
+set default_transaction_read_only = false;
+\c regression - - :master_port
+SELECT * FROM public.check_database_on_all_nodes($$altered_database!''2$$) ORDER BY node_type;
+
+\c regression - - :master_port
+set citus.log_remote_commands = true;
+set citus.grep_remote_commands = "%ALTER DATABASE%";
+alter database "altered_database!'2" owner to CURRENT_USER;
+set default_transaction_read_only = false;
+SELECT * FROM public.check_database_on_all_nodes($$altered_database!''2$$) ORDER BY node_type;
+
+set citus.enable_alter_database_owner to off;
+alter database "altered_database!'2" owner to test_owner_non_main_db;
+SELECT * FROM public.check_database_on_all_nodes($$altered_database!''2$$) ORDER BY node_type;
+alter database "altered_database!'2" owner to CURRENT_USER;
+SELECT * FROM public.check_database_on_all_nodes($$altered_database!''2$$) ORDER BY node_type;
+reset citus.enable_alter_database_owner;
+
+
+
```

Comment:
still seeing some https://github.com/citusdata/citus/pull/7563/#discussion_r1546491105

Reply:
I redesigned the sql file after some review comments. Those are the newly formed empty lines removed all of them